projects/vanilla/src/app/search/search-configurator.component.ts

```diff
@@ -18,11 +18,12 @@ export class SearchConfiguratorComponent implements OnDestroy {
    * The configuration from the config.ts file can be overridden by configuration from
    * the app configuration on the server
    */
-  public get metadata(): string[] {
-    const m = this.appService.app?.data?.metadata as any as MetadataConfig[] || METADATA_CONFIG;
-    return m.map(i => i.field);
+  public get metadataConfig(): MetadataConfig[] {
+    return this.appService.app?.data?.metadata as any as MetadataConfig[] || METADATA_CONFIG;

```

Comment:
Why do you need this ? In vanilla builder, all the configuration should be in the components config. There should be no need for METADATA_CONFIG or the appService.app.data.

Reply:
It can be removed, I've added it only for a handy feature so that if you add a metadata that's configured in METADATA_CONFIG or appService it prefills its configuration, but we can keep it simple and just make that everything is to configure from scratch

Reply:
For the metadata in the mini preview, you can already prefill the configuration by specifying it in the preview component configuration (maybe it's already the case?).
For the metadata in the results, I think its ok if you have to fill the configuration from scratch.

(the problem is that this extra configuration is confusing in vanilla builder, because users expect all the configuration to be defined in the components)

Reply:
Yeah I removed the input, the simpler the better!